tests/unit/CompareRaftMembers/DiffStreamOplogFilter.js

```diff
@@ -5,7 +5,7 @@ const { shuffle } = require('arsenal');
 
 const DiffStreamOplogFilter = require('../../../CompareRaftMembers/DiffStreamOplogFilter');
 
-const HTTP_TEST_PORT = 9090;
+const HTTP_TEST_PORT = 9091;
```

Comment:
a safer alternative may be to let the server:
1. bind to localhost only (avoid requiring extra permissions)
2. allocates a random port (i.e. specify port = 0)

this way, there will be no conflict, and the port to connect to can be retrieved by calling `server.address().port`, something like this:
```
  server = http.createServer();
  server.listen(':0');
  port = server.address().port
```